wiki/Tournaments/BruinGG/SoCal_Fall_Festival/en.md

```diff
@@ -0,0 +1,60 @@
+---
+tags:
+  - BruinGG
+  - California
+  - Los Angeles
+  - SFF
+  - southern California
+  - UCLA
+---
+
+# SoCal Fall Festival
+
+![SoCal Fall Festival banner](img/banner.jpg)
+
+The **SoCal Fall Festival** is an upcoming 1v1 [osu!standard](/wiki/Game_mode/osu!) tournament hosted by *BruinGG*<!-- TODO: link --> in collaboration with *Gaucho Gaming*<!-- TODO: link --> and [osu! UCI](/wiki/Organisations/osu!_UCI). This tournament brings together communities from University of California campuses, but anyone can join!
+
+In addition to the osu! tournament documented here, the SoCal Fall Festival will host casual games, competitive tournaments, panels, and an artists alley.
+
+## Links
+
+- **[Discord server](https://discord.gg/au7qpnk)**
+- [Forum thread](https://osu.ppy.sh/community/forums/topics/1168265)
+- [Rules (Google document)](https://docs.google.com/document/d/1HX3zr4dU8NlimMubDSiRFn8vP3DusvgVgG1WYiKGqmQ/edit?usp=sharing)
+
+### Related Discord servers
+
+- [BruinGG](https://discord.gg/bkxNBay)
+- [Gaucho Gaming](https://discord.gg/GauchoGaming)
+- [osu! UCI](https://discord.gg/qbZddFV)
+- [TAG at UCI](https://discord.gg/qKqCZXQ)
+
+## Schedule
+
+| Event | Timestamp (UTC-8) |
+| --: | :-- |
+| Registration | 2020-10-24 / 2020-11-07 (16:00) |
+| Swiss stage | 2020-11-07 (16:00) |
+| Round of 16 | 2020-11-07 (after swiss stage) or 2020-11-08 (16:00) |
+| Quarterfinals–Finals | 2020-11-08 (16:00) or 2020-11-08 (after Round of 16) |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 1 year of osu! supporter, Cooler Master CK552 with Gateron Red switches |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 8 months of osu! supporter, Cooler Master MM720 |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") ![Bronze crown](/wiki/shared/crown-bronze.png "4th place") | 6 months of osu! supporter |
```

Comment:
```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 1 year of osu! supporter tag, Cooler Master CK552 with Gateron Red switches |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 8 months of osu! supporter tag, Cooler Master MM720 |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place"), *4th place* | 6 months of osu! supporter tag |
```

Reply:
couldn't decide between this and 2 bronze crowns lol, they both look bad...